src/applications/dhp-connected-devices/components/ConnectedDevicesContainer.jsx

```diff
@@ -75,14 +81,17 @@ export const ConnectedDevicesContainer = () => {
         />
       </div>
       <h2>Devices you can connect</h2>
-      <div>
-        Choose a device type below to connect. You will be directed to an
-        external website and asked to enter your sign in information for that
-        device. When complete, you will return to this page on VA.gov.
-      </div>
-
       <div data-testid="devices-to-connect-section">
-        <DevicesToConnectSection connectedDevices={connectedDevices} />
+        {connectionAvailable ? (
```

Comment:
Hmm I'm not crazy about having to check if `connectionAvailable` is true twice (once here, once on line 42). Can we render the `Unable to connect health device` alert in an `else` statement on line 45? Then leave line 78 and below as it was before this PR? May have to move line 83 (`<h2>Devices you can connect</h2>`) up to line 40

Reply:
I moved this check into the DevicesToConnectSection component instead. The DevicesToConnectSection now expected two args: connectedDevices and connectionAvailable